client/.eslintrc

```diff
@@ -9,6 +9,7 @@
   "rules": {
     "no-debugger" : 1,
     "comma-dangle": 0,
+    "no-underscore-dangle": 0,
```

Comment:
I think we decided to keep this rule back when I was upgrading `eslint-config-airbnb`: https://github.com/weaveworks/scope/pull/2058

However, if you think `__WEAVEWORKS_CSRF_TOKEN` should be an exception, I'd much rather add `/* eslint no-underscore-dangle: 0 */` to `web-api-utils.js` than disable this rule globally.